src/coprocessor/codec/mysql/mod.rs

```diff
@@ -64,17 +64,25 @@ mod tests {
     #[test]
     fn test_parse_frac() {
         let cases = vec![
+            ("", 0, 0),
+            ("", 10, 0),
```

Comment:
Please add a non-zero test case for the `10u32.pow` branch, e.g. `("12", 5, 12000)`.

Reply:
`("9999999", 8, 99999990)` for this purpose. I have changed it to `("12", 5, 12000)`